templates/cli/src/index.js

```diff
@@ -86,12 +86,17 @@ async function runPrompts() {
     required: true,
   });
 
+  const authorGitHub = await input({
```

Comment:
Do we want to allow for an alternative link to another kind of profile?

Reply:
as in, in addition to the github, aka prompt them twice? or to make the prompt more general so they are not asked to put in a github profile link specifically, but any link?

Reply:
Right, any link in one prompt. Maybe suggest GitHub as an example.